src/main/java/org/amahi/anywhere/service/AudioService.java

```diff
@@ -144,15 +153,18 @@ private void setUpBus() {
     }
 
     private void setUpAudioPlayer() {
-        audioPlayer = new MediaPlayer();
-
-        audioPlayer.setAudioStreamType(AudioManager.STREAM_MUSIC);
-        audioPlayer.setWakeMode(this, PowerManager.PARTIAL_WAKE_LOCK);
-        audioPlayer.setVolume(1.0f, 1.0f);
+        audioPlayer =
+            ExoPlayerFactory.newSimpleInstance(
```

Comment:
`newSimpleInstance(Context, TrackSelector, LoadControl)` is deprecated, you may use `newSimpleInstance(RenderersFactory, TrackSelector, LoadControl)`.